web-lib-funcs.pl

```diff
@@ -266,6 +266,27 @@ sub html_escape
 return $tmp;
 }
 
+=head2 html_unescape(string)
```

Comment:
Aren't there way more entities (like the numeric ones) that covered here?

Reply:
> Aren't there way more entities (like the numeric ones) that covered here?

Yeap! This patch https://github.com/webmin/webmin/pull/1917/commits/8d66b62fcede2c174409ffb0a0a46ce3bcd6a67c will address it.